tests/reboot/reboot_test.go

```diff
@@ -60,7 +60,10 @@ var _ = Describe("{RebootOneNode}", func() {
 					}
 				})
 
-				Step(fmt.Sprintf("reboot app %s's node(s): %v", ctx.App.Key, nodesToReboot), func() {
+			}
+
+			// Reboot node and check driver status
+				Step(fmt.Sprintf("reboot node one at a time from the node(s): %v",  nodesToReboot), func() {
 					for _, n := range nodesToReboot {
```

Comment:
In the loop before this section, nodes for volumes of all apps is being collected. The reason for not using `node.GetWorkerNodes()`  in the reboot section  is because there could be nodes which do not have any volumes, like storage-less nodes.

Reply:
Changed the code to use node.GetWorkerNodes()